.changeset/bright-dogs-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": patch
+---
+
+Fix `DialogContent` spacing
```

Comment:
Maybe be a little more specific? So when user reads this, they could know where to look for the diff in their own app.